DigitalLearningSolutions.Web/Models/DelegateRegistrationData.cs

```diff
@@ -22,6 +22,8 @@ public DelegateRegistrationData(int? centreId, int? supervisorDelegateId = null,
         public string? Answer4 { get; set; }
         public string? Answer5 { get; set; }
         public string? Answer6 { get; set; }
+        public string? ProfessionalRegistrationNumber { get; set; }
+        public bool? HasProfessionalRegistrationNumber { get; set; }

```

Comment:
Fair enough. I was thinking that because it is just the same properties without any special editing-specific functionality (e.g. validation attributes) it would be fine to implement it elsewhere. But I now realise the purpose is for multiple view models to be able to be applied to that editing view, so it makes sense not to apply it here.